en/components/card.md

```diff
@@ -287,8 +287,122 @@ You can also justify the buttons so that they are layed out across the entire ax
+
+```scss
+$dark-card: igx-card-theme(
+    $palette: $dark-palette,
+    $background:#fd6bd93a,
+    $header-text-color:#f2f607,
+    $subtitle-text-color:#fa2509,
+    $content-text-color:#0a41f7
+);
+```
+
+>[!NOTE]
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please refer to [`Palettes`](../themes/palette.md) topic for detailed guidance on how to use them.
+
+#### Using Schemas
+
+Going further with the theming engine, you can build a robust and flexible structure that benefits from [**schemas**](../themes/schemas.md). A **schema** is a recipe of a theme.
+
+Extend one of the two predefined schemas, that are provided for every component, in this case - the [`dark-card`]({environment:sassApiUrl}/index.html#variable-_dark-card) schema:
+
+```scss
+// Extending the dark-card schema
+$dark-card-schema: extend($_dark-card,
+    (
+        background:(
+            igx-color:("secondary", 100)
+        ),
+        header-text-color:(
+            igx-color:("primary", 500)
+        ),
+        subtitle-text-color:(
+            igx-color:("primary", 500)
+        ),
+        content-text-color:(
+            igx-color:("primary", 700)
+        )
+    )
+);
+```
+
+In order to apply our custom schemas we have to **extend** one of the globals ([`light`]({environment:sassApiUrl}/index.html#variable-light-schema) or [`dark`]({environment:sassApiUrl}/index.html#variable-dark-schema)), which is basically pointing out the components with a custom schema, and after that add it to the respective component themes:
+
+```scss
+// Extending the global dark-schema
+$custom-dark-schema: extend($dark-schema,(
+    igx-card: $dark-card-schema
+));
+
+// Defining the dark-card theme with the global dark schema
+$dark-card: igx-card-theme(
+  $palette: $dark-palette,
+  $schema: $custom-dark-schema
+);
+```
+
+Don't forget to include the themes in the same way as it was demonstrated above.
+
+#### Demo
+
+<div class="sample-container loading" style="height: 486px">
+    <iframe id="card-style-iframe" seamless="" width="100%" height="100%" frameborder="0" data-src="{environment:demosBaseUrl}/layouts/card-styling-sample" class="lazyload no-theming">
+</iframe></div>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="card-style-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+### Summary
+In this article we covered a lot of ground with the card component. First, we created a very simple card with text content only. Then added some images to make the card a bit more appealing. We used some additional Ignite UI for Angular components inside our card, like avatars, buttons and icons, to enrich the experience and add some functionality. And finally, we changed the card's theme by setting some exposed theme colors, creating custom palletes and extending schemas. 
```

Comment:
Typo: **palletes** should be **palettes**

Reply:
Fixed